astrobee/survey_manager/survey_planner/src/survey_planner/command_astrobee.py

```diff
@@ -120,15 +120,15 @@ def get_ops_plan_path():
     # Check if the environment variable $ASTROBEE_OPS exists
     astrobee_ops_path = os.getenv("ASTROBEE_OPS")
     if astrobee_ops_path:
-        return os.path.join(astrobee_ops_path, "gds/plans")
+        return os.path.join(astrobee_ops_path, "dock_scripts/hsc/gds/plans")
```

Comment:
Obviously it's critical for this to work on the real robot, but I hope it can work in multi-robot software sim as well. Did you check it to see if the [instructions here](https://github.com/nasa/isaac/blob/52c5741ad5839590ee8072a1eaebcf3af8013837/astrobee/survey_manager/survey_planner/readme.md) still work?

This looks like it may be depending on an environment variable `$ASTROBEE_OPS`. If so, the sim instructions should probably be updated to explain how to set that. (Note that the `$OPS_REPO` variable name wasn't something I made up but rather comes from [older wiki instructions](https://babelfish.arc.nasa.gov/confluence/display/FFFSW/Running+GDS+in+Linux).

Note: I'm not saying this is broken, I just haven't had a chance to test it yet and want to make sure the issue is on your radar. Thanks!

Reply:
Yeah it still works because there are symlinks inside the ops repo. So doing `gds/plans` is the same as `dock_scripts/hsc/gds/plan`s, but to the robot, we only copy the folder `dock_scripts/hsc/`

Reply:
regarding the env variable name, that's only used if you want to overwrite the usual path. If you have GDS installed, that's used if no `OPS_REPO` defined